### DISCUSSION TITLE
Suggestions on how to train a subject that have text?

### DISCUSSION THREAD

**Discussion by ergosumdre** [TheLastBen/fast-stable-diffusion#489] [Q&A]
I'm trying to train the model on a series of images like this:
![crdtom(16)](https://user-images.githubusercontent.com/35677602/201544120-4ad7bcb4-4034-4965-9834-9e548f75cbd8.jpg)

However, when I render the images after training, there is always an issue with the characters. The image either has missing or extra characters. For example:
![tmpzexzdqws](https://user-images.githubusercontent.com/35677602/201544333-610119d2-5ec7-4969-ab29-8dcddb0569af.png)

Can anyone help suggest ways to improve the model?

**Comment by TheLastBen** [comment]
Text isn't something diffusion models are good at, even Dalle-2 isn't.

**Comment by giteeeeee** [comment]
Try generating the image with the 1.5 vae loaded. It improves text generation somewhat.

**Reply by ergosumdre** [reply]
Thank you! Can I bother you to ask how to do this? There isn't a place in the colab notebook to upload a vae. Sorry, I'm pretty new to this entire space.

**Reply by TheLastBen** [reply]
this colab has the latest vae included, no need to install it
